package.json

```diff
@@ -27,34 +27,33 @@
   },
   "homepage": "https://github.com/GoogleChrome/selenium-assistant#readme",
   "devDependencies": {
-    "chai": "^4.1.2",
-    "chromedriver": "^2.33.1",
-    "coveralls": "^3.0.0",
-    "eslint": "^4.9.0",
-    "eslint-config-google": "^0.9.1",
-    "express": "^4.16.2",
-    "firebase": "^4.5.2",
-    "geckodriver": "^1.9.0",
-    "glob": "^7.1.2",
+    "chai": "^4.2.0",
+    "chromedriver": "^75.1.0",
```

Comment:
Did you need to make this `*` so it works with future versions?

Reply:
Yeah, let me try that, along with doing `npm install` on Travis instead  of `npm ci` (so that the `package-lock.json` is hopefully ignored.)

Reply:
Actually, based on https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci, I think the best approach is to not use `package-lock.json` at all for this project.

I'm going to follow the suggestions in https://codeburst.io/disabling-package-lock-json-6be662f5b97d to disable it.

Reply:
Actually... maybe the best approach is to keep using `package-lock.json` but to add in a pre-test script that runs `npm install <driver>` (without `--save-dev`). I'm going to play around with that.